src/logreqres.c

```diff
@@ -198,6 +198,7 @@ size_t reqresAppendRequest(client *c) {
         !strcasecmp(cmd,"debug") ||
         !strcasecmp(cmd,"pfdebug") ||
         !strcasecmp(cmd,"lolwut") ||
+        (!strcasecmp(cmd,"sentinel") && argc > 1 && !strcasecmp(argv[1]->ptr,"simulate-failure")) ||
```

Comment:
Its tests have not been added yet. it can be removed, nothing special, just heard Guy say `both lists should be identical`
thinking about how to add tests to it later

Reply:
i had a discussion about that with Guy and concluded that we wanna designate the list in the C file just for command that don't really have replies (like MONITOR, DEBUG SEGFAULT, SUBSCRIBE).
and that for the rest we rather just have them being skipped in the python (like you did).
we added these changes to https://github.com/redis/redis/pull/12083 which i intend to merge first.
after that, we can take our time with this PR or another to add the necessary schemas and tests for sentinel.

meaning that when rebasing, you can drop this change and a few others.